khmer/_khmermodule.cc

```diff
@@ -4480,6 +4578,38 @@ static PyMethodDef khmer_hllcounter_methods[] = {
     {NULL} /* Sentinel */
 };
 
+static PyGetSetDef khmer_hllcounter_getseters[] = {
+    {
+        (char const *)"alpha",
+        (getter)hllcounter_getalpha, NULL,
+        (char const *)"alpha constant for this HLL counter.",
+        NULL
+    },
+    {
+        (char const *)"error_rate",
+        (getter)hllcounter_get_erate, (setter)hllcounter_set_erate,
+        (char const *)"Error rate for this HLL counter. "
+        "Can be changed prior to first counting, but becomes read-only after "
+        "that (raising AttributeError)",
+        NULL
+    },
+    {
+        (char const *)"ksize",
+        (getter)hllcounter_get_ksize, (setter)hllcounter_set_ksize,
+        (char const *)"k-mer size for this HLL counter."
+        "Can be changed prior to first counting, but becomes read-only after "
+        "that (raising AttributeError)",
+        NULL
+    },
+    {
+        (char const *)"counters",
+        (getter)hllcounter_getcounters, NULL,
+        (char const *)"Read-only internal counters.",
+        NULL
+    },
+    {NULL} /* Sentinel */
+};
```

Comment:
`khmer/_khmermodule.cc:4611:1: warning: deprecated conversion from string constant to ‘char*’ [-Wwrite-strings]` ?